spacehammer/lib/files.fnl

```diff
@@ -0,0 +1,92 @@
+(local {: contains?
+        : split
+        : some} (require :spacehammer.lib.functional))
+
+(fn file-exists?
+  [filepath]
+  "
+  Determine if a file exists and is readable.
+  Takes a file path string
+  Returns true if file is readable
+  "
+  (let [file (io.open filepath "r")]
+    (when file
+      (io.close file))
+    (~= file nil)))
+
+(fn copy-file
+  [source dest]
+  "
+  Copies the contents of a source file to a destination file.
```

Comment:
nit: `Copy the contents`, so it's the same tone as the rest of the comments (e.g. Determine x instead of Determines x)